### PR TITLE
Refacto timeline

### PR DESCRIPTION
**Pull request by freddidierRTE**
nothing in release note

### CODE REVIEW

File: ui/main/src/app/business/view/timeline/circles.spec.ts [hunk 1]
```diff
@@ -0,0 +1,260 @@
+/* Copyright (c) 2018-2023, RTE (http://www.rte-france.com)
```
Comment (quinarygio) [inline ui/main/src/app/business/view/timeline/circles.spec.ts:1]: Copyright for new files should be just for 2023

File: ui/main/src/app/business/view/timeline/circles.ts [hunk 2]
```diff
@@ -0,0 +1,246 @@
+/* Copyright (c) 2018-2023, RTE (http://www.rte-france.com)
```
Comment (quinarygio) [inline ui/main/src/app/business/view/timeline/circles.ts:1]: Copyright for new files should be just for 2023

File: ui/main/src/app/business/view/timeline/timeline.view.ts [hunk 3]
```diff
@@ -0,0 +1,120 @@
+/* Copyright (c) 2018-2023, RTE (http://www.rte-france.com)
```
Comment (quinarygio) [inline ui/main/src/app/business/view/timeline/timeline.view.ts:1]: Copyright for new files should be just for 2023

File: ui/main/src/app/business/view/timeline/xaxis.ts [hunk 4]
```diff
@@ -0,0 +1,189 @@
+/* Copyright (c) 2018-2023, RTE (http://www.rte-france.com)
```
Comment (quinarygio) [inline ui/main/src/app/business/view/timeline/xaxis.ts:1]: Copyright for new files should be just for 2023

**Review by quinarygio — CHANGES_REQUESTED** [review]
In realtime the hours labels are oblique, they used to be horizontal

File: ui/main/src/app/business/view/timeline/circles.spec.ts (hunk 1 above)
Comment (freddidierRTE) [reply ui/main/src/app/business/view/timeline/circles.spec.ts:1]: keep it as it contains code from a previous existing file

File: ui/main/src/app/business/view/timeline/timeline.view.ts (hunk 3 above)
Comment (freddidierRTE) [reply ui/main/src/app/business/view/timeline/timeline.view.ts:1]: keep it as it contains code from a previous existing file

File: ui/main/src/app/business/view/timeline/circles.ts (hunk 2 above)
Comment (freddidierRTE) [reply ui/main/src/app/business/view/timeline/circles.ts:1]: keep it as it contains code from a previous existing file

File: ui/main/src/app/business/view/timeline/xaxis.ts (hunk 4 above)
Comment (freddidierRTE) [reply ui/main/src/app/business/view/timeline/xaxis.ts:1]: keep it as it contains code from a previous existing file

**Pull request merged**